_posts/chall12/2017-05-22-caroline-turtle-hack.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+author: chall12
+title: "Caroline's Turtle Exercise"
+---
+
+
+Here's the program I'm embedding:
+
+
+<iframe src="https://trinket.io/embed/python/6c004e7fe9" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+
+Reflection:
+
+My idea for making a flower began on the first day of class when we learned to make circles. I first changed the background 
+color to light blue and created a rectangle along the base of the screen filled green to make grass. To begin the flower, I clumped two 
+circles together for each petal (it took me a while to get the size of the circle and their distances to work so that it didn't look 
+like a mass of yellow). Then I had to find the center of the flower to make the pollen cluster. Afterwords, I calculated the distance
+between the grass and the base of the flower to create the stem. It would have been smarter to create the stem first and draw the flower 
+over it, but I didn't think of that at first. Inserting the stem into the beginning of the code messed up the petal position, so 
+calculating its placement seemed like the easier alternative. Afterwords, I wanted to play with text some, so I added a few phrases 
+scattered around the flower in different colors.
+
+
+While this flower looks extremely simple, it actually took me over 2.5 hours to code. I had a really hard time with the petal placement, 
```

Comment:
The results show!  This is the precision I'm looking for.  Though, I hope you find you're getting faster as we go along.